frontend/dashboard/src/components/dashboard.tsx

```diff
@@ -0,0 +1,100 @@
+import { Container, 
+         Grid2 as Grid,
+         Card,
+         CardContent,
+         Typography,
+         CardActionArea
+} from '@mui/material';
+import React from 'react';
+
+const workflowsCard = (
+  <React.Fragment>
+    <CardActionArea href="/workflows">
+      <CardContent>
+        <Typography gutterBottom variant="h5" component="div">
+          Workflows
+        </Typography>
+        <Typography variant="body2" sx={{ color: 'text.secondary' }}>
+          Provides an overview of current and past workflows for 
+          a given visit.
+        </Typography>
+      </CardContent>
+    </CardActionArea>
+  </React.Fragment>
+)
+
+const templatesCard = (
+  <React.Fragment>
+    <CardActionArea href='/templates'>
+      <CardContent>
+        <Typography gutterBottom variant="h5" component="div">
+          Templates
+        </Typography>
+        <Typography variant="body2" sx={{ color: 'text.secondary' }}>
+          Provides an overview of all available workflowTemplates and Sensors.
+        </Typography>
+      </CardContent>
+    </CardActionArea>
+  </React.Fragment>
+)
+
+const documentationCard = (
+  <React.Fragment>
+    <CardActionArea href='https://diamondlightsource.github.io/workflows/docs' target='_blank'>
+      <CardContent>
+        <Typography gutterBottom variant="h5" component="div">
+          Documentation
+        </Typography>
+        <Typography variant="body2" sx={{ color: 'text.secondary' }}>
+          Full user documentation with how-tos, examples and tutorials.
```

Comment:
Our docs follow the [THER format](https://danielsieger.com/blog/2023/04/24/framework-for-better-documentation.html)

```suggestion
          Full user documentation with tutorials, how-tos, explanations and references.
```